src/rules/require-valid-changelog.ts

```diff
@@ -28,9 +32,13 @@ export default buildRule({
       });
       return pass();
     } catch (error) {
-      if (isErrorWithCode(error) && error.code === 'ERR_INVALID_JSON_FILE') {
+      if (
+        isErrorWithCode(error) &&
+        isErrorWithMessage(error) &&
+        error.code === 'ERR_INVALID_JSON_FILE'
+      ) {
         throw new Error(
-          'The package does not have a well-formed manifest. This is not the fault of the changelog, but this rule requires a valid package manifest.',
+          `${error.message}\n The package does not have a well-formed manifest. This is not the fault of the changelog, but this rule requires a valid package manifest.`,
```

Comment:
It seems like the error message explains why the package is not well-formed, so perhaps that should go afterward? Perhaps we can reword the message as a whole like so:

``` suggestion
          `Tried to get version from package manifest in order to validate changelog, but manifest is not well-formed (${error.message}).`,
```

What do you think?